simulation/src/range_likelihood.cpp

```diff
@@ -180,336 +317,384 @@ pcl::simulation::RangeLikelihood::RangeLikelihood (int rows, int cols, int row_h
-  glDeleteTextures (1, &depth_texture_);
-  glDeleteTextures (1, &color_texture_);
-  glDeleteTextures (1, &score_texture_);
-  glDeleteTextures (1, &score_summarized_texture_);
-  glDeleteTextures (1, &sensor_texture_);
-  glDeleteTextures (1, &likelihood_texture_);
-  glDeleteFramebuffers (1, &fbo_);
-  glDeleteFramebuffers (1, &score_fbo_);
-  glDeleteRenderbuffers (1, &depth_render_buffer_);
-  glDeleteRenderbuffers (1, &color_render_buffer_);
-
-  delete [] depth_buffer_;
-  delete [] color_buffer_;
-  delete [] score_buffer_;
+  glDeleteBuffers(1, &quad_vbo_);
+  glDeleteTextures(1, &depth_texture_);
+  glDeleteTextures(1, &color_texture_);
+  glDeleteTextures(1, &score_texture_);
+  glDeleteTextures(1, &score_summarized_texture_);
+  glDeleteTextures(1, &sensor_texture_);
+  glDeleteTextures(1, &likelihood_texture_);
+  glDeleteFramebuffers(1, &fbo_);
+  glDeleteFramebuffers(1, &score_fbo_);
+  glDeleteRenderbuffers(1, &depth_render_buffer_);
+  glDeleteRenderbuffers(1, &color_render_buffer_);
+
+  delete[] depth_buffer_;
+  delete[] color_buffer_;
+  delete[] score_buffer_;
 }
 
 double
-pcl::simulation::RangeLikelihood::sampleNormal (double sigma)
+pcl::simulation::RangeLikelihood::sampleNormal(double sigma)
 {
-  std::normal_distribution<> dist (0.0, sigma);
-  return (dist (rng));
+  std::normal_distribution<> dist(0.0, sigma);
+  return (dist(rng));
 }
 
 void
-pcl::simulation::RangeLikelihood::setupProjectionMatrix ()
+pcl::simulation::RangeLikelihood::setupProjectionMatrix()
 {
-  glMatrixMode (GL_PROJECTION);
-  glLoadIdentity ();
+  glMatrixMode(GL_PROJECTION);
+  glLoadIdentity();
 
   // Prepare scaled simulated camera projection matrix
-  float sx = static_cast<float> (camera_width_) / static_cast<float> (col_width_);
-  float sy = static_cast<float> (camera_height_) / static_cast<float> (row_height_);
-  float width = static_cast<float> (col_width_);
-  float height = static_cast<float> (row_height_);
-
-  float fx = camera_fx_/sx;
-  float fy = camera_fy_/sy;
-  float cx = camera_cx_/sx;
-  float cy = camera_cy_/sy;
+  float sx = static_cast<float>(camera_width_) / static_cast<float>(col_width_);
+  float sy = static_cast<float>(camera_height_) / static_cast<float>(row_height_);
+  float width = static_cast<float>(col_width_);
+  float height = static_cast<float>(row_height_);
+
+  float fx = camera_fx_ / sx;
+  float fy = camera_fy_ / sy;
+  float cx = camera_cx_ / sx;
+  float cy = camera_cy_ / sy;
   float m[16];
-  float z_nf = (z_near_-z_far_);
-
-  m[0] = 2.0f*fx/width;  m[4] = 0;              m[ 8] = 1.0f-(2*cx/width);      m[12] = 0;
-  m[1] = 0;              m[5] = 2.0f*fy/height; m[ 9] = 1.0f-(2*cy/height);     m[13] = 0;
-  m[2] = 0;              m[6] = 0;              m[10] = (z_far_+z_near_)/z_nf;  m[14] = 2.0f*z_near_*z_far_/z_nf;
-  m[3] = 0;              m[7] = 0;              m[11] = -1.0f;                  m[15] = 0;
-  glMultMatrixf (m);
+  float z_nf = (z_near_ - z_far_);
+
+  m[0] = 2.0f * fx / width;
```

Comment:
Maybe reorder these so the pattern is visible again. Instead of this method, use the initializer list 
```c++
float m[16] = {2.f*fx/width, 0,             0, 0,
               0,            2.f&fy/height, 0, 0...
```